build/Plugin.nuspec

```diff
@@ -0,0 +1,88 @@
+         </group>
+         <group targetFramework="netstandard1.1">
+		    <dependency id="Common.Logging.Core" version="3.4.1" exclude="Build,Analyzers" />
+			<dependency id="Newtonsoft.Json" version="10.0.3" exclude="Build,Analyzers" />
+         </group>
+		 <group targetFramework="netstandard1.3">
+		    <dependency id="System.IO.Pipes" version="4.3.0" exclude="Build,Analyzers" />
+			<dependency id="EdjCase.JsonRpc.Client" version="1.3.1" exclude="Build,Analyzers" />
+			<dependency id="System.Net.WebSockets.Client" version="4.3.2" exclude="Build,Analyzers" />
+         </group>
+		 <group targetFramework="netstandard2.0">
+		    <dependency id="System.IO.Pipelines" version="4.5.1" exclude="Build,Analyzers" />
+         </group>
+         <group targetFramework="netcoreapp2.1">
+			<dependency id="System.IO.Pipes" version="4.3.0" exclude="Build,Analyzers" />
+			<dependency id="System.Net.WebSockets.Client" version="4.3.2" exclude="Build,Analyzers" />
+         </group>
+       </dependencies>
+   </metadata>
+   <files> 
+       <!--Net 1.1-->
+	   <!--<file src="..\src\JsonRpcSharp.Client\bin\Release\net11\JsonRpcSharp.Client.*" target="lib\net11" />-->
+	   
+       <!--Net 3.5-->
+	   <file src="..\src\JsonRpcSharp.Client\bin\Release\net35\JsonRpcSharp.*" target="lib\net35" />
+	   
+       <!--Net 4.5.1-->
+       <file src="..\src\JsonRpcSharp.Client\bin\Release\net451\JsonRpcSharp.*" target="lib\net451" />
+	   <file src="..\src\JsonRpcSharp.RpcClient\bin\Release\net451\JsonRpcSharp.*" target="lib\net451" />
+	   
+	   <!--Net 4.6.1-->
+       <file src="..\src\JsonRpcSharp.IpcClient\bin\Release\net461\JsonRpcSharp.*" target="lib\net461" />
+	   <file src="..\src\JsonRpcSharp.WebSocketClient\bin\Release\net461\JsonRpcSharp.*" target="lib\net461" />
+	   
+	   <!--netstandard1.1-->
+       <file src="..\src\JsonRpcSharp.Client\bin\Release\netstandard1.1\JsonRpcSharp.*" target="lib\netstandard1.1" />
+	   <file src="..\src\JsonRpcSharp.RpcClient\bin\Release\netstandard1.1\JsonRpcSharp.*" target="lib\netstandard1.1" />
+	   
+	   <!--netstandard1.3-->
+       <file src="..\src\JsonRpcSharp.IpcClient\bin\Release\netstandard1.3\JsonRpcSharp.*" target="lib\netstandard1.3" />
+	   <file src="..\src\JsonRpcSharp.UnixIpcClient\bin\Release\netstandard1.3\JsonRpcSharp.*" target="lib\netstandard1.3" />
+	   <file src="..\src\JsonRpcSharp.WebSocketClient\bin\Release\netstandard1.3\JsonRpcSharp.*" target="lib\netstandard1.3" />
+	   
+	   <!--netstandard2.0-->
+       <file src="..\src\JsonRpcSharp.TcpClient\bin\Release\netstandard2.0\JsonRpcSharp.*" target="lib\netstandard2.0" />
+	   
+       <!--Net Core 2.1-->
+       <file src="..\src\JsonRpcSharp.IpcClient\bin\Release\netcoreapp2.1\JsonRpcSharp.*" target="lib\netcoreapp2.1" />
+	   <file src="..\src\JsonRpcSharp.WebSocketClient\bin\Release\netcoreapp2.1\JsonRpcSharp.*" target="lib\netcoreapp2.1" />
+   </files>
```

Comment:
hey @winstongubantes there is a mix of tabs and spaces in this file, can you fix please?

Reply:
also can you rename the file from `Plugin.nuspec` to `JsonRpcSharp.nuspec`? thanks

Reply:
already updated the file